tools/grab_spotify_saved_tracks.py

```
@@ -0,0 +1,175 @@
+import hashlib
+
+# Load environment variables
+load_dotenv()
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+
+def setup_spotify():
```

Ruff (0.8.2)

Missing return type annotation for public function `setup_spotify`

(ANN201)